pmmdemo/README.md

```diff
@@ -50,6 +50,34 @@ To prepare for a successul launch of pmmdemo infrastructure, please follow the i
 1. Run `direnv allow` to inject the variables from `.envrc` file to your environment.
 2. Run `terraform init` to initialize your terraform state.GitHub Pull Requests and Issues
 
+## List of servers
+
+We use default hostname in table below: `demo.local` but if you use non-default terraform profile then your hostname will contain your profile name. For example, `test.local` for test profile.
+
+### Databases
+
+| Name                       | Hostnames                                                 |
```

Comment:
Looks good, but the problem is we are dealing with `ServiceNames` in the interface, and here we talk about hostnames. How do they relate?